environments/production/tofu/backend.tf

```
@@ -1,13 +1,25 @@
+variable "s3_backend_endpoint" {
+  type        = string
+  description = "radosgw address without protocol or path e.g. leafcloud.store"
+  default = "https://api.dl.acrc.bris.ac.uk:6780"
+}
+
+# tflint-ignore: terraform_required_version
 terraform {
   backend "s3" {
-    endpoint = "https://api.dl.acrc.bris.ac.uk:6780"
-    key = "terraform.tfstate"
-    bucket = "slurm-production-tofu-state"
-    # Ceph doesn't use the region, but OpenTofu requires it
-    region = "not-used-but-required"
-    skip_region_validation = "true"
-    # The STS API doesn't exist for Ceph
-    skip_credentials_validation = "true"
-    use_path_style = "true"
+    endpoint = var.s3_backend_endpoint
+    bucket   = "slurm-production-tofu-state"
+    key      = "terraform.tfstate"
+
+    # Reginon is required but not used in radosgw:
+    region                 = "dummy"
+    skip_region_validation = true
+
+    # Normally STS is not configured in radosgw:
+    skip_credentials_validation = true
+
+    # Enable path-style S3 URLs (https://<HOST>/<BUCKET> instead of https://<BUCKET>.<HOST>)
+    # may or may not be required depending on radosgw configuration
+    use_path_style = true
   }
-}
+}
```

Missing blank line at end of file.

```suggestion
}

```